recipe/conda_build_config.yaml

```diff
@@ -669,6 +671,8 @@ tbb_devel:
   - 2020
 thrift_cpp:
   - '0.14.1'
+tinyxml2:
+  - 9.0
```

Comment:
Should we add `8` here and also add a migration at the same time or are you sure that everything is already migrated?

Reply:
Agreed - have done both. There are quite a few packages left that need migration.